env/initialize.sh

```diff
@@ -213,11 +213,15 @@ initialize_chromedriver() {
   fi
   log_message "Installing chromedriver..."
   rm -rf ./chromedriver
-  # Determine the currently installed version of Chrome. Leave only the numbers,
-  # e.g., 72.0.3626.0.
-  local chrome_version=$(google-chrome --version | cut -f 3 -d ' ')
-  # Download Chromedriver. As of now, the maintainers guarantee that there's a
-  # matching Chromedriver artifact for every publicly released Chrome version.
+  # Determine the version of the latest available stable chromedriver.
+  # The link is taken from here https://github.com/GoogleChromeLabs/chrome-for-testing#other-api-endpoints.
+  local chromedriver_version_url="https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_STABLE"
```

Comment:
If the version fetched here mismatches the installed version of Chrome, I think Chromedriver will fail.

Reply:
So if we're to follow this approach, we'd need to install this `Chrome for Testing`. Might be doable - I haven't checked the details.

I've also commented on the Issue with some thoughts about an alternative solution.